models/bamf_mr_brain_tumor/meta.json

```diff
@@ -0,0 +1,206 @@
+        ]
+      },
+      {
+        "label": "T1ce Segmentation",
+        "type": "Segmentation",
+        "description": "Tumor Segmentation of T1ce MRI scan",
+        "classes": [
+          "NECROSIS",
+          "EDEMA",
+          "ENHANCING_LESION"
+        ]
+      },
+      {
+        "label": "T2 Segmentation",
+        "type": "Segmentation",
+        "description": "Tumor Segmentation of T2 MRI scan",
+        "classes": [
+          "NECROSIS",
+          "EDEMA",
+          "ENHANCING_LESION"
+        ]
+      },
+      {
+        "label": "FLAIR Segmentation",
+        "type": "Segmentation",
+        "description": "Tumor Segmentation of FLAIR MRI scan",
+        "classes": [
+          "NECROSIS",
+          "EDEMA",
+          "ENHANCING_LESION"
+        ]
+      }
+    ],
+    "model": {
+      "architecture": "U-net",
+      "training": "supervised",
+      "cmpapproach": "3D"
+    },
+    "data": {
+      "training": {
+        "vol_samples": 1251
+      },
+      "evaluation": {
+        "vol_samples": 45
+      },
+      "public": true,
+      "external": true
+    }
+  },
+  "details": {
+    "name": "AIMI MR Brain Tumor",
+    "version": "1.0.0",
+    "devteam": "BAMF Health",
+    "authors": [
+      "Soni, Rahul",
+      "McCrumb, Diana",
+      "Murugesan, Gowtham Krishnan",
+      "Van Oss, Jeff"
+    ],
+    "type": "nnU-Net (U-Net structure, optimized by data-driven heuristics)",
+    "date": {
+      "code": "17.10.2023",
+      "weights": "28.08.2023",
+      "pub": "23.10.2023"
+    },
+    "cite": "Murugesan, Gowtham Krishnan, Diana McCrumb, Mariam Aboian, Tej Verma, Rahul Soni, Fatima Memon, and Jeff Van Oss. The AIMI Initiative: AI-Generated Annotations for Imaging Data Commons Collections. arXiv preprint arXiv:2310.14897 (2023).",
+    "license": {
+      "code": "MIT",
+      "weights": "CC BY-NC 4.0"
+    },
+    "publications": [
+      {
+        "title": "The AIMI Initiative: AI-Generated Annotations in IDC Collections",
+        "uri": "https://arxiv.org/abs/2310.14897"
+      }
+    ],
+    "github": "https://github.com/bamf-health/aimi-brain-mr"
+  },
+  "info": {
+    "use": {
+      "title": "Intended Use",
+      "text": "To perform tumor segmentation in Brain MRI scans, you need four MRI modalities—T1-weighted, T1 with contrast enhancement (T1ce), T2-weighted, and FLAIR—each providing critical information about brain anatomy and tumor characteristics. The data should be organized in an input directory with subfolders for each modality, containing the respective DICOM files. The model processes these inputs to produce three segmentation maps for necrosis, edema, and enhancing tumor regions. These outputs are vital for clinical applications like treatment planning, monitoring tumor progression, and prognosis, as well as for research projects focused on developing new algorithms, identifying imaging biomarkers, and integrating automated segmentation into clinical workflows"
```

Comment:
[...] The data should be organized in separated directories per instance  (e.g., named by a unique sample or patient identifyer) with one subfolders for each of the four input modalites, containing the respective DICOM files (namely "t1", "t1ce", "t2", and "fair"). [...]

Reply:
Thank you. I have updated these text.